### PR TITLE
File icon handled and added for mime type analytics.

### PR DESCRIPTION
**Pull request by surinder-tsys**
Handled and added the file icon for the following MimeTypes:

- application/vnd.oasis.opendocument.formula
- application/vnd.oasis.opendocument.formula-template
- application/vnd.oasis.opendocument.graphics
- application/vnd.oasis.opendocument.graphics-template

- [ ] Tests written, or not not needed

### CODE REVIEW

File: app/src/main/res/drawable/file_analytics.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<!--
```
Comment (tobiasKaminsky) [inline app/src/main/res/drawable/file_analytics.xml:1]: Please update license. Or was this really done by you and can be GPL3+?

Comment (surinder-tsys) [reply app/src/main/res/drawable/file_analytics.xml:1]: Are you planning to use the same icon? If yes, then I am not sure what exactly license we have to use it here.

Comment (AndyScherzinger) [reply app/src/main/res/drawable/file_analytics.xml:1]: However there seemed to be no code detecting this mime-type leading to this icon ultimately being used, no? @surinder-tsys @tobiasKaminsky

**Review by AndyScherzinger — APPROVED** [review]
Fine by me! 🚀 

Thanks @surinder-tsys 👍

**Pull request merged**